indent/crystal.vim

```diff
@@ -632,4 +632,4 @@ endfunction
 let &cpo = s:cpo_save
 unlet s:cpo_save
 
-" vim:set sw=2 sts=2 ts=8 et:
```

Comment:
This line is a special line recognized by vim to handle automatically set some options for this file, it shouldn't be changed, see `:h modeline` in vim.